### DISCUSSION TITLE
impossible acces to graphql url with digitalOcean droplet

### DISCUSSION THREAD

**Discussion by desdamo** [directus/directus#6637] [Archive]
Hello 
everything is in the title. i cant access to graphql i tried /graphql ,  /graphql/system, /ProjectName/graphql/ , etc ...
I'm a newbie with Directus. Can you help me please ?

**Comment by rijkvanzanten** [comment]
The endpoint is `/graphql/`. 

> i cant access to graphql

What do you mean by this? Are you getting an error? 404? Keep in mind that Directus doesn't ship with any GraphQL explorer tool ootb
